src/index.ts

```diff
@@ -25,5 +28,5 @@ export { default as Timeline } from './Timeline/index.js'
 export { default as UserIcon } from './UserIcon/index.js'
 export { default as Spinner } from './Animations/Spinner/index.js'
 
-export { CheckedIcon, InProgressIcon, PadlockIcon }
+export { CheckedIcon, InProgressIcon, PadlockIcon, LeftArrowIcon, RightArrowIcon, RefreshIcon }
```

Comment:
Are we getting to the point that we should have an `Icons` export that contains these? This might get messy